frontend/src/lib/components/Playlist/Playlist.tsx

```diff
@@ -44,6 +44,7 @@ export type PlaylistProps<T> = {
     onChangeSections?: (activeKeys: string[]) => void
     'data-attr'?: string
     activeItemId?: string
+    controls?: JSX.Element | null
```

Comment:
playlist is used in multiple places so keeping this row relatively abstract as "controls" rather than just calling it sorting